spec/features/favorites/index_spec.rb

```diff
@@ -87,6 +87,22 @@
     visit "/favorites"
     expect(page).to have_content("(0) Favorited Pets")
     expect(page).to have_content("You have no pets favorited!")
-    
+  end
+
+  it "can remove all Favorites from Favorites Page" do
+    visit "/pets/#{@pet_1.id}"
+    click_on "Favorite Pet"
+
+    visit "/pets/#{@pet_2.id}"
+    click_on "Favorite Pet"
+
+    visit '/favorites'
+
+    expect(page).to have_link("Remove All Pets From Favorites")
```

Comment:
Probably don't need this because the click on will error out if there is no button